embedded-graphics/src/mock_display.rs

```diff
@@ -238,6 +238,34 @@ where
         self.pixels[i as usize] = color;
     }
 
+    /// Get the affected area of the display.
+    pub fn affected_area(&self) -> Rectangle {
```

Comment:
This should have a changelog entry, because it's public.
```suggestion
    /// Returns the area that was affected by drawing operations.
    pub fn affected_area(&self) -> Rectangle {
```